fetch_worker.go

```diff
@@ -49,9 +49,26 @@ func newFetchWorker(db *sqlx.DB, data chan fetchData, config fetchWorkerConfig)
 }
 
 func (w *fetchWorker) fetchStartEnd(ctx context.Context) (err error) {
-	w.start = w.config.lastPosition.Snapshots[w.config.table].LastRead
-	w.end, err = w.getMaxValue(ctx)
-	return err
+	lastRead := w.config.lastPosition.Snapshots[w.config.table].LastRead
```

Comment:
Should never be missing, the whole worker fails if so.

Reply:
Sorry, I think I might have misunderstood your comment. If `w.config.table` is missing, yeah, the fetch worker obviously shouldn't work.
If `w.config.lastPosition.Snapshots[w.config.table]` is missing it will start from 0. That would happen when the pipeline is restarted with additional specified tables.
I don't see the problem @hariso, could you please elaborate? I'm happy to correct the code, but I'm lost on what to fix here.